docs/sections/user_guide/cli/drivers/orog/help.out

```diff
@@ -12,6 +12,10 @@ Optional arguments:
 
 Positional arguments:
   TASK
+    files_copied
+      Files copied for run
+    files_hardlinked
+      Files hard-linked for run
```

Comment:
For some reason, orog only supported `files_linked` previously, and I _think_ that should be up to the user on how to stage the files.